lib/fog/compute/google/models/operations.rb

```diff
@@ -5,26 +5,27 @@ class Operations < Fog::Collection
         model Fog::Compute::Google::Operation
 
         def all(filters = {})
-          if filters["zone"]
-            data = service.list_zone_operations(filters["zone"]).body
-          elsif filters["region"]
-            data = service.list_region_operations(filters["region"]).body
-          else
-            data = service.list_global_operations.body
-          end
-          load(data["items"] || [])
+          data = if filters.zone
```

Comment:
Given that we're rewriting compute and it's >9k lines, this would be a good time to do that.